package.json

```diff
@@ -1,6 +1,16 @@
 {
-  "name": "visible-website",
+  "name": "visible-wordpress",
   "version": "1.0.0",
+  "scripts": {
+    "export-db": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp db export /data/database.sql\"",
+    "import-db": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp db import /data/database.sql\"",
+    "update-plugins": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp plugin update --all\"",
+    "update-themes": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp theme update --all\"",
+    "plugin-install": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp plugin install $PLUGIN\"",
+    "plugin-update": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp plugin update $PLUGIN\"",
+    "theme-update": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp theme update $THEME\"",
+    "theme-install": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp theme install $THEME\""
```

Comment:
Lines 11-12 need to be ran the same as above, only using `THEME`...

`THEME=some-theme npm run theme-install`